app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -95,9 +141,28 @@ class MainActivity : ComponentActivity() {
 
     @Composable
     fun ProfileMenu(){
-        Box(Modifier.fillMaxWidth(), contentAlignment = Alignment.CenterEnd) {
-            Icon(imageVector = Icons.Filled.Menu, contentDescription = "")
+        val context = LocalContext.current
+        Box(
+            modifier = Modifier
+                .background(color = Color.LightGray)
+                .fillMaxWidth()
+                .padding(5.dp),
+                contentAlignment = Alignment.CenterStart)
+
+        {
+            Row()
+            {
+                IconButton(
+                    onClick = { Toast.makeText(context, "Menu Clicked", Toast.LENGTH_LONG).show() }
+                ) {
+                    Icon(imageVector = Icons.Filled.Menu, contentDescription = "")
+                }
```

Comment:
I did a lot of refactoring of the profileMenu() since it was blending into the spinner and didn't seem like there were separate elements there.  I removed the logon button that was glaring in the top left of the screen and gave the box containing the burger menu a grey color to differentiate between the section with the dropdown icon and text fields.  Additionally, the menu icon was originally just an Icon.  I changed it to be an IconButton, so you can add an onClick to open a menu of initiate a pop-up menu.